### PR TITLE
Improve UI page

### PR DESCRIPTION
**Pull request by domdinicola**
- allow to schedule the sync task through the ui
- updated template (extend django admin)

### CODE REVIEW

File: data_api/ui/views.py [hunk 1]
```diff
@@ -31,3 +36,21 @@ def import_org_view(request):
         'form': form,
         'feedback': feedback
     })
+
+
+class BasicTaskAPIView(APIView, metaclass=ABCMeta):
```
Comment (czue) [inline data_api/ui/views.py]: This requires Py3

**Review by czue — COMMENTED** [review]
This looks really nice. 

However, I'm getting this error in the console when I load the page (it still loads, but thinking we should fix this?)

```
DEBUG Exception while resolving variable 'title' in template 'ui/import_org.html'.
Traceback (most recent call last):
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/base.py", line 882, in _resolve_lookup
    current = current[bit]
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/context.py", line 87, in __getitem__
    raise KeyError(key)
KeyError: 'title'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/base.py", line 888, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'title'
```
Also, we either need to make this Py2 compatible or wait till we've rolled out Py3 before merging.

**Comment by domdinicola** [comment]
let's wait for django2, I don't think we will have those warnings

**Comment by domdinicola** [comment]
Screenshot_2019-08-06_10-24-20

**Comment by domdinicola** [comment]
@czue pr ready for review

**Review by czue — APPROVED** [review]
Code looks good. From a UX perspective it took me a while to figure out that the "sync latest data" button actually syncs the entire warehouse. I wonder if we could change the text a bit to make that more clear? Like "Sync Entire Warehouse" or something?

I think one of the problems is that these two things are only semi-related, so having them lumped together on the same page is a little funny. But I think we can roll with it for now.

**Comment by domdinicola** [comment]
@czue amended PR, my idea was to make the page a more generic admin/management page. What do you think about that?

- changed label
- admin actions: added messages
- admin actions: add full sync

**Review by czue — APPROVED** [review]
:+1: sounds good

**Pull request merged**